cmd/bacalhau/create.go

```diff
@@ -168,3 +182,39 @@ var createCmd = &cobra.Command{
 
 	},
 }
+
+var templateCmd = &cobra.Command{
```

Comment:
I don't see the need for this command when you have an `--output-template` parameter on the actual docker run?